pandas/tests/copy_view/test_interp_fillna.py

```diff
@@ -336,6 +340,11 @@ def test_fillna_inplace_ea_noop_shares_memory(
         # arrow is immutable, so no-ops do not need to copy underlying array
         assert np.shares_memory(get_array(df, "b"), get_array(view, "b"))
     else:
-        assert not np.shares_memory(get_array(df, "b"), get_array(view, "b"))
+        # MaskedArray can actually respect inplace=True
+        assert np.shares_memory(get_array(df, "b"), get_array(view, "b"))
```

Comment:
I think this one and the elif before are equal now

Reply:
will update